advocacy_docs/partner_docs/CommVaultGuide/04-ConfiguringTheIntegration.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: 'Configuring the Integration'
+description: 'Walkthrough on configuring Commvault Backup & Recovery'
+---
+
+Implementing Commvault Backup & Recovery with EDB Postgres Advanced Server requires the following components:
+
+- EDB Postgres Advanced Server
+- Commvault Backup & Recovery software
+
+## Prerequisites
+
+- A running EDB Postgres Advanced Server instance.
+- Commvault Backup & Recovery installed.
+- EDB Postgres Advanced Server application path and library directory path (e.g. c:\Program Files\edb\as13\bin and c:\Program Files\edb\as13\lib).
+- The login credentials used to access the EDB Postgres Advanced Server database.
+- EDB Postgres Advanced Server archive log directory configured.
+
+## Configure Commvault Backup & Recovery for EDB Postgres Advanced Server
+
+### Core Setup Wizard
+1. Run the Core Setup Wizard from Commvault Backup & Recovery's Command Centre from the Machine where Commvault Backup & Recovery is installed. The wizard helps to set up a disk storage pool and to modify the server backup plan according to your requirements.
+2. Setup Storage Pool/Disk Storage: From the Welcome page, click Let's get started. On the Disk tab, in the Name box, enter a name for the storage pool.
+3. In the MediaAgent box, accept the default value.
+4. Click Local on the radio button for Type.
+5. In the Backup location box, click Browse to assign a path where backups will be stored.
+6. To enable deduplication on the storage, move the Use deduplication toggle key to  the right, and in the Deduplication DB location box, browse to select the path to  the deduplication database.
+7. To move to the next setup option, click Save
+
+<p align="center">
+  <img  alt="Setup Storage Pool/Disk Storage" src="Images/DiskStorageConf.png">
+</p>
+
+8. Creating a Server Backup Plan in Core Setup: In order to create a server backup plan in Core Setup, define where the data is stored and how often must the data undergo a backup.
+9. In the Plan name box,type the name of the plan e.g EPAS Server Plan.
+10. In the Backup destinations section, select the Storage and Retention Period.
+11. In the RPO section, select the Backup frequency and Start time for the backup to start as per the backup frequency.
+
+<p align="center">
+  <img  alt="Creating a Server Backup Plan in Core Setup" src="Images/BackupPlanConf.png">
+</p>
+
+### Setup EDB Postgres Advanced Server
+1. From the navigation pane, go to Protect > Databases.
```

Comment:
Step 1
 1. From the Navigation pane of the Commvault Backup & Recovery's Command Center, go to Protect > Databases.